examples/interactive_scripting.py

```diff
@@ -1,5 +1,6 @@
 import numpy as np
 import napari
+from napari._qt.threading import thread_worker
```

Comment:
now that #1122 has merged will this go in `napari.qt`? Probably a good place for it

Reply:
will move